templates/pricing/infra.html

```diff
@@ -11,31 +11,29 @@
 
 <section id="ua-infra" class="p-strip--suru-topped is-bordered">
   <div class="u-fixed-width">
-    <h1>Open Infrastructure support & security</h1>
-    <p>Ubuntu Advantage for Infrastructure covers all aspects of open infrastructure - Ubuntu, KVM, OpenStack, Kubernetes, Docker, storage, networking, Ceph, SWIFT, SDN, logging, alerting and monitoring.</p>
-    <ul class="p-inline-list">
-      <li class="p-inline-list__item">
-        <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
-      </li>
-      <li class="p-inline-list__item">
-        <a href="/advantage" class="p-button--neutral">Purchase a subscription</a>
-      </li>
-    </ul>
+    <h1>Support and security pricing</h1>
+    <!-- TODO Find copy -->
```

Comment:
Did you find it? :P

Reply:
As I understand it it's not written yet. so... no!